src/Uno.Foundation/Collections/ValueSet.cs

```diff
@@ -1,34 +1,71 @@
-﻿#nullable enable
-using System;
-using System.Collections;
-using System.Collections.Generic;
+﻿using System.Collections;
 
 namespace Windows.Foundation.Collections;
 
-public sealed partial class ValueSet : IPropertySet, IObservableMap<string, object?>, IDictionary<string, object?>, IEnumerable<KeyValuePair<string, object?>>
+/// <summary>
+/// Implements a map with keys of type String and values of type Object.
+/// </summary>
+public sealed partial class ValueSet :
+	IDictionary<string, object>,
+	IEnumerable<KeyValuePair<string, object>>,
+	IObservableMap<string, object>,
+	IPropertySet
 {
-	private readonly Dictionary<string, object?> _dictionary = new();
+	private readonly Dictionary<string, object> _properties = new Dictionary<string, object>();
```

Comment:
Why did you remove the nullability on this one?

Reply:
Unintentional, fixed!